### ISSUE TITLE
Correction needs to be done at footer across all localized K8s site

### ISSUE THREAD

**Issue by aj11anuj** [kubernetes/website#43068]
**This is a Feature Request**

**What would you like to be added?**
As due to a recent PR merged #42962, now all localized K8s sites need to make this updation by changing the word "Home" at footer to "Documentation" across all pages. See below Original K8s site:

![Screenshot (681)](https://github.com/kubernetes/website/assets/116108975/fddf77f5-76c0-4664-852b-b3208ed21cff)

**Comment by satyampsoni** [comment]
I would be sending the PR once the triage is clear.
/assign

**Comment by Gauravpadam** [comment]
/remove area/web-development
It's more of a doc modification imo

**Comment by Gauravpadam** [comment]
Thanks for raising this issue @aj11anuj!

**Comment by jihoon-seo** [comment]
@satyampsoni it seems that you want to unassign yourself, so let me help with that.
/unassign @satyampsoni

**Comment by Ank206** [comment]
Is this issue still Open and can i contribute in it?

**Comment by aj11anuj** [comment]
Yes, this issue is still open.. although it doesn't have `triage/accepted` label

**Comment by zilmarr** [comment]
I'd like to work on the PT version if possible.

**Comment by Gauravpadam** [comment]
Sure @zilmarr, Go ahead and propose a fix

**Comment by IbukiMnw** [comment]
I'd like to do Japanese part.
/assign

**Comment by mengjiao-liu** [comment]
The ZH version has finished at https://github.com/kubernetes/website/pull/43172.
/remove-language zh

**Comment by Okabe-Junya** [comment]
It looks that some languages has already finished, so let me remove some labels ;)

/remove-language ja de es fr id it ko ru uk vi hi

and now IbukiMnw is not working on this issue (#43547 is already merged)

/unassign  IbukiMnw

**Comment by dkarczmarski** [comment]
hi :) i can see that this change was made in commit : https://github.com/kubernetes/website/commit/7b721061eebf397ae488a1f5df348d0c2649906a for polish language , so can we remove `language/pl` label ?

**Comment by reylejano** [comment]
As this issue relates to a stylistic change , I think there may be other areas for localization that may be more important 

/label area/localization
/remove-area web-development

**Comment by divya-mohan0209** [comment]
The footer has been removed, currently, across all localisations. 
/triage accepted
/close
